openstates/wi/bills.py

```diff
@@ -329,36 +329,29 @@ def add_senate_votes(self, vote, url):
             return
 
         doc = lxml.html.fromstring(html)
+        trs = doc.xpath('//table[@class="senate"]/tbody/tr')
 
-        yes_count = no_count = other_count = 0
+        vote_types = ['yes', 'no', 'other']
```

Comment:
The third type is `not voting`, right?

Reply:
I am not really sure, absent, excused were not given, are they are taking all others in ```not voting``` only, that's why I thought maybe other would be right to use here. But yeah ```not voting``` is mentioned here. I should with ```not voting``` then?

Reply:
Yes, if the state labels them as "Not Voting" then we should use that Open Civic Data category.